### DISCUSSION TITLE
Remove Help (?) button

### DISCUSSION THREAD

**Discussion by kovrom** [Budibase/budibase#3878] [Help] [answered]
First of all, amazing project! Thank you!
Quick question, is it possible somehow to hide the Help (?) button? It links to the discussion page here on github and if anything more confusing than helpful for a regular user. Alternatively is it possible to link it to a different url (maybe internal FAQ, etc). Thank you.

**Comment by mjashanks** [comment, marked as answer]
I think I see what you mean - it only really makes sense if viewing in developer mode.

I.e. it doesn't make screen on the regular user's app portal as below?

![image](https://user-images.githubusercontent.com/3524181/147633096-b8341f28-b33c-4a4a-b693-4393dcb75b1f.png)

**Reply by kovrom** [reply]
@mjashanks Yup. That's it. It makes sense to have it in developer/administrator mode. But it should be possible to either hide it or customize url for it on Login Screen and for logged in regular users.

**Reply by dadlMilestone** [reply]
Did this ever get implemented? I am happy to see this question answered, but I still have same issue as OP.

**Reply by ryzam** [reply]
@mjashanks , if possible to hide?
